docs/docsite/rst/user_guide/get_started_inventory.rst

```diff
@@ -0,0 +1,103 @@
+.. _get_started_inventory:
+
+*********************
+Building an inventory
+*********************
+
+Inventories organize managed nodes in centralized files that provide Ansible with system information and network locations.
+Using an inventory file, Ansible can manage a large number of hosts with a single command.
+Inventories also help you use Ansible more efficiently by reducing the number of command-line options you need to specify.
+For example, inventories usually contain the SSH user so you do not need to include the ``-u`` flag when running Ansible commands.
+
+In the previous section, you added managed nodes directly to the ``/etc/ansible/hosts`` file.
+Now let's create an inventory file that you can add to source control for flexibility, reuse, and for sharing with other users.
+
+Complete the following steps: 
+
+#. Open a terminal window on your control node.
+#. Create a new inventory file named ``inventory.yaml`` in any directory and open it for editing.
+#. Add a new group for your hosts then specify the IP address or FQDN of each managed node with the ``ansible_host`` field.
+   The following example adds the IP addresses of three virtual machines in KVM: 
+
+   .. literalinclude:: yaml/inventory_example_vms.yaml
+      :language: yaml
+
+#. Verify your inventory.
+   If you created your inventory in a directory other than your home directory, specify the full path with the ``-i`` option.
+
+   .. code-block:: bash 
+
+      ansible-inventory -i inventory.yaml --list
+
+#. Ping the managed nodes in your inventory.
+   In this example, the group name is ``virtualmachines`` which you can specify with the ``ansible`` command instead of ``all``.
+
+   .. code-block:: bash 
+
+      ansible virtualmachines -m ping -i inventory.yaml
+
+   .. literalinclude:: ansible_output/ping_inventory_output.txt
+      :language: text
+
+Congratulations! You have successfully built an inventory.
+
+Best practices when building inventories
```

Comment:
```suggestion
Tips for building inventories
```

Reply:
...some folks are very against the idea of 'best practices' as a concept overall.